src/aat/resources/features/F-142 - Get Linked Case V2 External/CreateCases/F-142_CreateLinkedCase_StandardCaseLink.td.json

```diff
@@ -0,0 +1,90 @@
+{
+  "_guid_": "F-142_CreateLinkedCase_StandardCaseLink",
+  "_extends_": "F-142_CreateCase_MasterCaseType__Base",
+
+  "request": {
+    "body": {
+      "data": {
+        "caseNameHmctsInternal": "Name internal: ${[scenarioContext][customValues][generateUUID]}",
+
+        "caseLinks": [
+          {
+            "id": "52837798-42c6-43cc-98f6-0895fdba4961",
```

Comment:
Can this not use `${[scenarioContext][customValues][generateUUID]}` to generate a UUID rather than hardcoding it ?

Reply:
Or is it because we cant reference that value when asserting the id in the response ?

Reply:
The latter. Although it would only become an issue in those matching TD files that create cases with mulitple links or multipe reasons for a link as I couldn't use `[[ANYTHING_PRESENT]]`; I would need the ID to be present for the `expectedResponse` assertion.